docs/source/international_languages.md

```diff
@@ -139,6 +139,9 @@ as spoken in Mexico:
    Once you've been added to this list, you can add others translators from your
    workgroup.
 
+1. On the [`i18n rocket channel`](https://chat.hyperledger.org/channel/i18n)
```

Comment:
Is a bulletpoint really needed here?

Reply:
I took the bullet point out. I was using the same structure, but this way is ok too.